src/store/singleFilm/logic.js

```diff
@@ -11,22 +14,27 @@ const singleFilmLogic = createLogic({
     try {
       const { data } = await httpClient.get(`/movie/${filmId}`);
 
+      const norm = normalize(data, Movie);
+
       const {
         data: { crew, cast },
       } = await httpClient.get(`/movie/${filmId}/credits`);
 
       const {
         data: { backdrops },
       } = await httpClient.get(`/movie/${filmId}/images`);
-
-      const aboutFilm = {
-        ...data,
+      const { movie, genres } = norm.entities;
+      const response = {
+        response: { ...data },
```

Comment:
You don't need to store response and id. You can grab id from router, and all movie details from database state